### PR TITLE
2445: Update translations

### PR DESCRIPTION
**Pull request by ztefanie**
### Short description
Update translations

### Proposed changes
- [x] Fix errors in translations.json
- [x] Decide for a solution to handle shelter translations
- [x] Add recurring events translations
- [x] Send constants to translators
- [ ] Import new translations

### Resolved issues

Fixes: #2445 

---

### CODE REVIEW

File: translations/translations.json [hunk 1]
```diff
@@ -7587,7 +7587,7 @@
       "hostInformation": "Angaben zum Anbietenden",
       "contactInformation": "Kontaktdaten",
       "shelterButton": "Anschauen",
-      "bed_one": "{{count}} Bett",
+      "bed_one": "1 Bett",
       "bed_other": "{{count}} Betten",
```
Comment (steffenkleinle) [inline translations/translations.json:7591]: Should it be 
```suggestion
      "bed_other": "{{count}} Betten",
```
or 
```suggestion
      "bed_other": "{{beds}} Betten",
```

**Review by steffenkleinle — COMMENTED** [review]
Would be nice if you could wait for a PR for just the translations of #2031 before forwarding the constants for translation.

File: translations/translations.json [hunk 2]
```diff
@@ -7671,8 +7671,8 @@
       "hostInformation": "Údaje o nabízejícím",
       "contactInformation": "Kontaktní údaje",
       "shelterButton": "Zobrazit",
-      "bed": "1 postel",
-      "beds": "{{beds}} postele",
+      "bed_one": "1 postel",
+      "bed_other": "{{beds}} postele",
```
Comment (steffenkleinle) [inline translations/translations.json:7675]: ~~This has to be `{{count}}` actually, could you please change all occurrences?
https://www.i18next.com/translation-function/plurals~~

Edit: Nvm, we'll remove the shelter constants anyway.

**Comment by ztefanie** [comment]
Closed, see new translations in https://github.com/digitalfabrik/integreat-app/pull/2560

**Comment by steffenkleinle** [comment]
@ztefanie is the branch still needed or can it be deleted?

**Comment by ztefanie** [comment]
@steffenkleinle lets keep it until our Portuguese translations are imported correctly.

**Pull request closed without merging**